internal/runbits/requirements/dependencies.go

```diff
@@ -0,0 +1,217 @@
+			packageVersion = source.Version
+			break
+		}
+	}
+
+	// Process new build plan's requirements into something we can easily compare with.
+	newArtifacts, err := buildplan.NewArtifactListing(newBuildResult.Build, false, r.Config)
+	if err != nil {
+		return errs.Wrap(err, "Unable to create artifact listing for new build plan")
+	}
+	newArtifactMap, err := newArtifacts.RuntimeClosure()
+	if err != nil {
+		return errs.Wrap(err, "Unable to compute runtime closure for new build plan")
+	}
+
+	// Determine the package's direct and indirect dependencies.
+	var directDependencies []*artifact.Artifact
+	numDependencies := make(map[artifact.ArtifactID]int)
+	totalDependencies := 0
+	for _, artf := range newArtifactMap {
+		if artf.Name != packageName {
+			continue
+		}
+		directDependencies = computeDependencies(&artf, &newArtifactMap, false)
+		directDependencies = filterDependencies(directDependencies, &oldRequirements, showUpdatedPackages)
+		totalDependencies = len(directDependencies)
+		for _, dep := range directDependencies {
+			indirectDependencies := computeDependencies(dep, &newArtifactMap, true)
+			indirectDependencies = filterDependencies(indirectDependencies, &oldRequirements, showUpdatedPackages)
+			numDependencies[dep.ArtifactID] = len(indirectDependencies)
+			totalDependencies += len(indirectDependencies)
+		}
+		break
+	}
+
+	pg.Stop(locale.T("progress_success"))
+	if len(directDependencies) == 0 {
+		return nil
+	}
+
+	// List additional dependencies.
+	r.Output.Notice("") // blank line
+
+	localeKey := "additional_dependencies"
+	if len(directDependencies) < totalDependencies {
+		localeKey = "additional_total_dependencies"
+	}
+	r.Output.Notice(locale.Tr(localeKey,
+		packageName, packageVersion, strconv.Itoa(len(directDependencies)), strconv.Itoa(totalDependencies)))
+
+	// A direct dependency list item is of the form:
+	//   ├─ name@version (X dependencies)
+	// or
+	//   └─ name@oldVersion → name@newVersion (Updated)
+	// depending on whether or not it has subdependencies, and whether or not showUpdatedPackages is
+	// `true`.
+	for i, dep := range directDependencies {
+		if i > maxListLength {
+			break
+		}
+		prefix := "├─"
+		if i == len(directDependencies)-1 || i == maxListLength {
+			prefix = "└─"
+		}
+
+		version := ""
+		if dep.Version != nil {
+			version = *dep.Version
+		}
+
+		subdependencies := ""
+		if numSubs := numDependencies[dep.ArtifactID]; numSubs > 0 {
+			subdependencies = fmt.Sprintf(" ([ACTIONABLE]%s[/RESET] dependencies)", strconv.Itoa(numSubs)) // intentional leading space
+		}
+
+		item := fmt.Sprintf("[ACTIONABLE]%s@%s[/RESET]%s", dep.Name, version, subdependencies) // intentional omission of space before last %s
+		if oldVersion, exists := oldRequirements[fmt.Sprintf("%s/%s", dep.Namespace, dep.Name)]; exists && version != "" && oldVersion != version {
+			item = fmt.Sprintf("[ACTIONABLE]%s@%s[/RESET] → %s (%s)", dep.Name, oldVersion, item, locale.Tl("updated", "Updated"))
```

Comment:
```suggestion
			item = fmt.Sprintf("[ACTIONABLE]%s@%s[/RESET] → %s (%s)", dep.Name, oldVersion, item, locale.Tl("updated", "updated"))
```

Consistent casing with "dependencies".